### DISCUSSION TITLE
Show an indicator for signed commits

### DISCUSSION THREAD

**Discussion by screendriver** [jesseduffield/lazygit#1512] [Ideas]
In other tools like for example [GitKraken](https://www.gitkraken.com) there is an indicator that a commit was signed and valid. It looks like this for example:

<img width="436" alt="Screenshot 2021-10-14 at 06 49 55" src="https://user-images.githubusercontent.com/149248/137254102-df304ed8-b4d1-491b-845d-832ceee9590c.png">

The small green paper with a pen is the indicator that it signed and valid (green). If it's not valid but still signed it would be grey.

It would be great if I could see something similar in `lazygit`.

**Comment by simon-abbott** [comment]
I second this. I want to always sign all my commits, but occasionally I need to temporarily turn it off and if I forget to turn it back on I can make an unsigned commit without realizing it. Having an indicator in the commits list would be amazing. Perhaps a green 'S' for signed and red 'U' for unsigned between the initials and the message?
